besu/src/main/java/org/hyperledger/besu/cli/subcommands/storage/TrieLogHelper.java

```diff
@@ -357,5 +351,30 @@ static void printCount(final PrintWriter out, final TrieLogCount count) {
         count.total, count.canonicalCount, count.forkCount, count.orphanCount);
   }
 
+  static void importTrieLog(
+      final BonsaiWorldStateKeyValueStorage rootWorldStateStorage,
+      final Path dataDirectoryPath,
+      final Hash trieLogHash) {
+    final String trieLogFile =
+        dataDirectoryPath.resolve(DATABASE_PATH).resolve(trieLogHash.toString()).toString();
+
+    var trieLog = readTrieLogsFromFile(trieLogFile);
```

Comment:
nit: not sure how important this is so feel free to ignore it doesn't make sense. But having the format in RLP to match the storage in the database trielog column family might make comparison easier without the extra encoding of the IdentityMap. Depends on the purpose of this tool.

Reply:
I'd lean towards a more readable format rather than binary as it's hard to work with. Agree RLP is a more useful middle ground than custom binary (if that's what this is).
e.g. I wonder if this could be used to import test data...compare with RlpBlockImporter.java and JsonBlockImporter

Reply:
I'd rather have us to extend the current feature to allow the export to RLP reusing this feature if necessary, it isn't quite simple to do that right now and it would impact the sub-command prune logic as well.